### PR TITLE
Hifi glimmer

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
This replaces the default `handlebars` implementation

### CODE REVIEW

File: tests/dummy/app/controllers/application.js [hunk 1]
```diff
@@ -9,9 +12,14 @@ export default class ApplicationController extends Controller {
   hbsExample = `
   {{!-- app/components/panel.hbs --}}
 
-  {{#if this.isAdmin}}
-    <Panel::Admin />
-  {{else}}
-    <Panel::NonAdmin />
-  {{/if}}`;
+  <Panel @user={{@name}} {{modifier 1 true}}>
```
Comment (NullVoxPopuli) [inline tests/dummy/app/controllers/application.js:15]: I changed this because I needed to see if I had configured things correctly

**Comment by rwwagner90** [comment]
@NullVoxPopuli so does this work for both glimmer and handlebars now?

**Comment by NullVoxPopuli** [comment]
Yup!

**Comment by NullVoxPopuli** [comment]
I would have just updated the base implementation, but the Prism folks are not interested in code authored in modern JavaScript and we're unwilling to transpile to their browser targets

**Comment by NullVoxPopuli** [comment]
demo (without ember) : https://prismjs-glimmer.nullvoxpopuli.com/

**Review by rwwagner90 — APPROVED** [review]

**Comment by Turbo87** [comment]
@rwwagner90 @NullVoxPopuli is there a way to opt out of this new dependency for apps that don't need it?

**Comment by NullVoxPopuli** [comment]
Why wouldn't you need it? 🤔 

The default handlebars syntax is incorrect

**Comment by Turbo87** [comment]
> The default handlebars syntax is incorrect

I'm mostly displaying Rust code on crates.io and I don't need the handlebars highlighter at all there (see https://github.com/rust-lang/crates.io/blob/5a76a330f8b2050cfe206478666dd3a4b1f206dc/ember-cli-build.js#L11-L28)

**Comment by NullVoxPopuli** [comment]
ah ok. well, tree-shaking should have you covered -- no import, no bundle?

**Comment by Turbo87** [comment]
> no import, no bundle?

it is unconditionally imported in `addon/index.js` though...

**Comment by NullVoxPopuli** [comment]
But you don't need to import that file,

**Comment by Turbo87** [comment]
> If you omit importing it, then wouldn't tree shaking eliminate all that?

no, because it's in the `addon` tree. maybe embroider would change that some day, but we're far away from being able to use embroider due to several incompatibilities.

**Comment by rwwagner90** [comment]
@NullVoxPopuli @Turbo87 can one of you perhaps submit a PR that conditionally includes it via a config or perhaps switch to dynamic imports?

**Pull request merged**